### ISSUE TITLE
http://localhost:8080/  , noVNC error !!!

### ISSUE THREAD

**Issue by bhatrana** [pieper/SlicerDockers#7]
@pieper  Hi there, I am also trying to make a 3D Slicer image in Docker using your repo. I used your master branch to replicate the same version 5.0.3, no issue on image build but there is an issue on connection to X11. Any suggestions ?

![image](https://github.com/pieper/SlicerDockers/assets/79718549/d42c774c-12df-479b-bbe4-edfc2ba327eb)

**Comment by pieper** [comment]
No, I haven't seen that - did you try running my image from Dockerhub?  It should still be working and you could exec into it and see what's different from your build.

My guess is that one of the dependency packages has changed (noVNC or x11vnc or something).  I believe the Dockerfile refers to the heads of the main repo branches but maybe specifying an older version hash would work.  Or updating things to work with the current I guess.

**Comment by bhatrana** [comment]
@pieper  pull from docker hub works but build from github doesn't. The error was shown above.  I checked the specific version of noVNC used in docker hub (it was 1.3) and the current is 1.5 . I switched to 1.3 but it still didn't work.

**Comment by pieper** [comment]
Okay, we'll, it would be great if you could look into what's different.  I haven't touched these docker files in a while, but I have used the noVNC / x11vnc approach in [these scripts](https://github.com/pieper/SlicerMachines) and it's been working well.

You may have to look into the noVNC source code for ideas.  From the error message linked above it seems to have something to do with the clipboard and maybe you can just disable that feature.
